### PR TITLE
feat: Implement Leaders Slice for Tablet

### PR DESCRIPTION
**Pull request by ValGeorgiev**
Implemet the Leaders Slice for Tablet 
Portrait: 
image

Landscape:
image

### CODE REVIEW

File: packages/slice-layout/src/templates/leaders/index.js [hunk 1]
```diff
@@ -1,14 +1,38 @@
...
+      <Column
+        style={styles.container}
+        tiles={[renderLeader1, renderLeader2, renderLeader3]}
+      />
+    );
+  }
+
+  return (
+    <View style={styles.container}>
+      <View style={styles.columnItems}>{renderLeader1()}</View>
```
Comment (tuncaulubilge) [inline packages/slice-layout/src/templates/leaders/index.js]: I'm not sure about this ordering.

In web, the middle tile seems to be leader1, so this is ordered as leader2 - leader1 - leader3.

@aronshamash might be able to verify this via render

Comment (aronshamash) [reply packages/slice-layout/src/templates/leaders/index.js]: this (the current render) ordering is correct.

File: packages/edition-slices/src/slices/leaders/index.js [hunk 2]
```diff
@@ -16,27 +17,53 @@ const renderHead = publicationName => (
...
+  );
 
+  return (
+    <Context.Consumer>
+      {({ publicationName }) => (
+        <View style={styles.container}>
+          {renderHead(publicationName)}
+          <ResponsiveSlice
+            renderHuge={renderHugeSlice}
+            renderMedium={renderSmallSlice}
```
Comment (tuncaulubilge) [inline packages/edition-slices/src/slices/leaders/index.js]: shouldn't we renderHude in renderMedium?

Comment (ValGeorgiev) [reply packages/edition-slices/src/slices/leaders/index.js]: Tablet portrait is like mobile, thats why renderMedium and renderSmall are with the same values

Comment (tuncaulubilge) [reply packages/edition-slices/src/slices/leaders/index.js]: Same tile, different layout, got it! 👍

**Pull request merged**